crates/ide/src/inlay_hints.rs

```diff
@@ -983,6 +988,7 @@ struct C;
 fn main() {
     let c = A(B(C))
         .into_b() // This is a comment
+        // This is another comment
         .into_c();
 }
```

Comment:
this comment actually doesn't affect the test as the last line, the `.into_c();` doesn't trigger chaining hints given its the last call. The comment should instead be above the `.into_b()` call.

Reply:
It affects the test in the following way:

- Without the added comment, two chaining hints are generated
- With the added comment and the current implementation, only one chaining hint is generated
- With the added comment and the modified implementation, two chaining hints are generated again (i.e., this is how I propose it should be)

Reply:
Oh I misunderstood the problem, I thought the comment affected the following lines. Yes then you are correct 👍 Was a bit confused as nothing changed in the test output(expected the TextRanges to change)